src/forms/currentExpenses.js

```diff
@@ -262,7 +262,7 @@ const ExpensesFormContent = function ({ client, time, setClientProperty }) {
       <wrapper>
         <FormHeading>Child Support</FormHeading>
         <IntervalColumnHeadings type={type}/>
-        <CashFlowRow {...sharedProps} generic={'ChildSupportPaidOut'}> LEGALLY OBLIGATED Child support paid out </CashFlowRow>
+        <CashFlowRow {...sharedProps} generic={'ChildSupportPaidOut'}> <strong>Legally Obligated</strong> child support paid out </CashFlowRow>
```

Comment:
'Legally Obligated child support paid out' should be 'Legally obligated child support'